src/neuroconv/datainterfaces/ecephys/neuralynx/neuralynxdatainterface.py

```diff
@@ -118,3 +83,109 @@ def __init__(self, folder_path: FolderPathType, sampling_frequency: float = None
         """
 
         super().__init__(folder_path=folder_path, sampling_frequency=sampling_frequency, verbose=verbose)
+
+
+def get_filtering_multi_channel(neo_reader) -> List[str]:
```

Comment:
```suggestion
def extract_filtering_metadata(neo_reader) -> List[str]:
```
I am confused about the multi channel here, are there some single channel  readers out there?

Reply:
Yes, in the old `spikeextractors` version there was also an `get_filtering_single_channel`. I renamed the method.